packages/libs/user-datasets/src/lib/Service/index.ts

```diff
@@ -1,29 +1,61 @@
-import { mapValues } from 'lodash';
-
 import { WdkService } from '@veupathdb/wdk-client/lib/Core';
+import { EpicDependencies } from '@veupathdb/wdk-client/lib/Core/Store';
+import { ActionThunk } from '@veupathdb/wdk-client/lib/Core/WdkMiddleware';
+
+import { UserDatasetApi, VDI_SERVICE_BASE_URL } from './api';
+
+const vdiCompatibilityFlag = '__IS_VDI_COMPATIBLE_SERVICE';
 
-import { userDatasetsServiceWrappers } from './UserDatasetWrappers';
-import {
-  ServiceConfig as UserDatasetUploadServiceConfig,
-  makeUserDatasetUploadServiceWrappers,
-} from './UserDatasetUploadWrappers';
+export type VdiCompatibleWdkService = ReturnType<typeof wrapWdkService>;
+
+export type ServiceConfig = {
+  datasetImportUrl: string;
+  fullWdkServiceUrl: string;
+};
 
 export function wrapWdkService(
-  serviceConfig: UserDatasetUploadServiceConfig | undefined,
+  serviceConfig: ServiceConfig | undefined,
   wdkService: WdkService
 ) {
-  const wrappersToInclude =
-    serviceConfig == null
-      ? userDatasetsServiceWrappers
-      : {
-          ...userDatasetsServiceWrappers,
-          ...makeUserDatasetUploadServiceWrappers(serviceConfig),
-        };
+  const vdiService = new UserDatasetApi(
+    { baseUrl: VDI_SERVICE_BASE_URL },
+    wdkService
+  );
 
   return {
+    [vdiCompatibilityFlag]: null,
```

Comment:
How about setting this to `vdiCompatibilityFlag` or `true`, just so it isn't a "falsey" value?